docs/user-guide/cli-mqplugin.md

```diff
@@ -25,24 +31,78 @@ Before you install the plug-in, meet the software requirements in [Software requ
 Use one of the following methods to install or update the plug-in:
 
 - [Installing plug-ins from an online registry](cli-installplugins.md#installing-plug-ins-from-an-online-registry)
-
 - [Installing plug-ins from a local package](cli-installplugins.md#installing-plug-ins-from-a-local-package)
 
 ## Creating a user profile
 
 After you install the plug-in, create an MQ profile. An MQ profile is recommended to issue commands to the MQ resource. MQ profiles contain your host, port, user name, and password for the IBM MQ REST API server of your choice. You can create multiple profiles and switch between them as needed.
 
-**Follow these steps:**
-1.  Install the IBM MQ Plug-in for Zowe CLI.
+Specify your plug-in profile and connection details in the `zowe.config.json` configuration file.
+
+### Creating plug-in profiles using a configuration file
+
+When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the MQ plug-in and then issue a `zowe config` command **[correct?]**, the command creates an entry for an MQ profile in your `zowe.config.json` file.
+
+Alternatively, you can create an MQ profile manually by adding a section that contains the configuration details to your `zowe.config.json` configuration file.
+
+#### Creating an MQ profile with a command
+
+1.  Install the IBM MQ Database Plug-in for Zowe CLI.
 2.  Create an MQ profile:
 
     ```
     zowe config init
     ```
-3.  Set the port number to your MQ port:
+3.  If using a non-standard port, set the port number to your MQ connection: **[is this step correct/applicable to MQ?]**
```

Comment:
@adam-wolfe - please respond

Reply:
I don't think we have a default port for MQ, so I think this is must be specified. @zFernand0

Reply:
You are right that there is no default port for MQ, and I believe we will need to add the `includeInTemplate` flag for the vNext version so that users are automatically prompted for it.